src/plugins/plot/stackedPlot/StackedPlot.vue

```diff
@@ -159,9 +160,9 @@ export default {
 
         exportJPG() {
             this.hideExportButtons = true;
-            const plotElement = this.$refs.plotContainer;
+            const plotElement = this.$el;
```

Comment:
plotContainer wasn't resolving, so switched to just $el. there's probably a more elegant way to do this

Reply:
this was preventing StackedPlots from exporting images by the way. this seemed broken in master as well.